CONTRIBUTING.md

```diff
@@ -309,9 +309,9 @@ flag. Below is a list of the available arguments and their purpose:
 
 | Build Argument | Default Value | Other Value(s) | Purpose |
 | --- | --- | --- | --- |
-| `CUDA_VERSION` | 10.0 | 10.1, 10.2 | set CUDA version |
-| `LINUX_VERSION` | ubuntu16.04 | ubuntu18.04 | set Ubuntu version |
-| `CC` & `CXX` | 5 | 7 | set gcc/g++ version; **NOTE:** gcc7 requires Ubuntu 18.04 |
+| `CUDA_VERSION` | 11.0 | 11.1, 11.2.2 | set CUDA version |
```

Comment:
There are few instances of 10.0 usage in this file.
https://github.com/galipremsagar/cudf/blob/remove_10.x/CONTRIBUTING.md#L163
https://github.com/galipremsagar/cudf/blob/remove_10.x/CONTRIBUTING.md#L285

Reply:
Done